service/core/kafka/client/src/main/resources/reference.conf

```diff
@@ -18,6 +18,9 @@ lagom.broker.kafka {
 
   client {
     default {
+      # how long should we wait when retrieving the last known offset
+      offset-timeout = 5s
```

Comment:
Introduced this timeout to keep symmetry with `ReadSideActor` but specially because there was no timeout 😅